.vib/phpmyadmin/goss/phpmyadmin.yaml

```diff
@@ -0,0 +1,41 @@
+command:
```

Comment:
I think we should also add a test for this section of the postunpack logic:

```
php_conf_set upload_max_filesize "$PHP_DEFAULT_UPLOAD_MAX_FILESIZE"
php_conf_set post_max_size "$PHP_DEFAULT_POST_MAX_SIZE"
php_conf_set memory_limit "$PHP_DEFAULT_MEMORY_LIMIT"
```

Reply:
I agree, especially for phpMyAdmin's upload requirements. I will skip checking `memory_limit` for now.

Reply:
Added